uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/my-leave/apis/entitlement-api_test.js

```diff
@@ -0,0 +1,55 @@
+define([
+  'leave-absences/shared/modules/apis',
+  'leave-absences/shared/apis/entitlement-api',
```

Comment:
Isn't `leave-absences/shared/modules/apis` already required by `leave-absences/shared/apis/entitlement-api`? If yes, there is no point of requiring it here as well